biogrid/libs/biogrid-simulator/src/lib/bioenergy-source/solar-panel.spec.ts

```diff
@@ -28,7 +28,8 @@ describe('tests for the BioEnergySource', () => {
       efficiency = 0.125,
       latitude = 0,
       area = 10,
-      date = new Date('07/20/2020');
+      date = new Date();
```

Comment:
what does this mean for your implementation? if users only get data for the past 7 days, what happens if they enter a date range from earlier?

Reply:
They will get an error back, there is an issue #80 which references this and that we need to restrict on the frontend

Reply:
Since you're limiting the date range to 1 week, maybe you can just remove that input at some point and default to the last full day?

Reply:
Ok sure, I added that to the issue